test/ideas.list.js

```diff
@@ -542,4 +542,146 @@ describe('read lists of ideas', () => {
       });
     });
   });
+
+  describe('GET /ideas?filter[commentedBy]=user0,user1,user2', () => {
+    let user0,
+        user2,
+        user3,
+        user4;
+    // create and save testing data
+    beforeEach(async () => {
+      const data = {
+        users: 6,
+        tags: 6,
+        verifiedUsers: [0, 1, 2, 3, 4],
+        ideas: [[{}, 0], [{}, 0],[{}, 1],[{}, 2],[{}, 2],[{}, 2],[{}, 3]],
+        userTag: [
```

Comment:
Is `userTag` and `ideaTags` necessary here?

Reply:
I thought we stated we are leaving it if it not disturbing.